### DISCUSSION TITLE
OSError: [Errno 8] Exec format error: '/home/pi/.local/share/undetected_chromedriver/undetected_chromedriver'

### DISCUSSION THREAD

**Discussion by valnurat** [ultrafunkamsterdam/undetected-chromedriver#1925] [General]
Hi.
I have installed the undetected chromedriver on my raspberry from [https://github.com/ultrafunkamsterdam/undetected-chromedriver](url) by doing this: pip install git+https://github.com/ultrafunkamsterdam/undetected-chromedriver.git

After I run the below code, I get OSError: [Errno 8] Exec format error: '/home/pi/.local/share/undetected_chromedriver/undetected_chromedriver'

Any one know why?
Br

import undetected_chromedriver as uc

options = uc.ChromeOptions()
options.add_argument('--headless')  # Runs the browser in headless mode.
options.add_argument('--no-sandbox')
options.add_argument('--disable-dev-shm-usage')

driver = uc.Chrome(options=options)

driver.get("https://www.google.com")
print(driver.title)

driver.quit()

**Comment by 0000vt** [comment]
I believe the driver isnt made for the Raspebrry pi ARM chip, Sounds like a problem like this one: https://stackoverflow.com/questions/53927815/oserror-errno-8-exec-format-error-using-chromedriver-with-selenium-and-linux

Perhaps downloading an ARM version and patching it could help.
